include/INA138.hpp

```diff
@@ -0,0 +1,49 @@
+#ifndef LVSS_INA138_HPP
+#define LVSS_INA138_HPP
+
+#include "EVT/dev/LCD.hpp"
```

Comment:
Use angle bracket includes for everything
```suggestion
#include <EVT/dev/LCD.hpp>
```
Also, you definitely don't need to include all these files. Remove the ones you don't need